packages/generator/src/generate.ts

```diff
@@ -65,24 +45,26 @@ const generate = async (props: GenerateProps) => {
       drawEmbeddedPage({ page, embeddedPage, embedPdfBox });
       for (let l = 0; l < keys.length; l += 1) {
         const key = keys[l];
-        const schema = schemas[j];
+        const schema = template.schemas[j];
         const templateSchema = schema[key];
         const input = inputObj[key];
-        const fontSetting = { font, pdfFontObj, fallbackFontName };
-
-        await drawInputByTemplateSchema({
-          input,
-          templateSchema,
-          pdfDoc,
-          page,
-          fontSetting,
-          inputImageCache,
-        });
+
+        if (!templateSchema || !input) {
+          continue;
+        }
+
+        // TODO ↓ If a Custom Schema is defined, it can undergo custom rendering processes by merging it with the renderer object.
+        // const renderer = Object.assign(buildInRenderer, options.schema);
+        const rendererObj = buildInRenderer[templateSchema.type];
```

Comment:
is the `builtInRenderer` more like `rendererRegsitry` or possibly `compositeRegistry`?  Is it only built-in renderers or will all of them be in this array?

Also, I can't see `builtInRenderer` actually declared or exported anywhere.

Reply:
It is more like rendererRegistry. 
In the future We allow users to add their own rendering processes to this registry using options.